platform/cog-platform-drm.c

```diff
@@ -46,6 +46,11 @@ struct buffer_object {
     uint32_t fb_id;
     struct gbm_bo *bo;
     struct wl_resource *buffer_resource;
```

Comment:
The intention of the PR seems to be replacing `buffer_resource` with `export.resource`, right? In that case, the `buffer_resource` member should be removed here.

Reply:
This pointer is used for the purposes of caching existing imported buffers. It's used for comparison in the `drm_buffer_for_resource()`.

The `export` struct below is used for holding references to whatever object has been presented and needs subsequent release. Holding separate references here makes it cleaner to test and release these resources, since e.g. for the `wpe_fdo_shm_exported_buffer` export both this object as well as the `wl_resource` would be set, but only the first one would have to be released.